pkg/domain/process.go

```diff
@@ -0,0 +1,70 @@
+				m.RespHeaderBytesTotal.WithLabelValues(serviceID, serviceName, datacenter, domain).Add(float64(stats.RespHeaderBytes))
+				m.StatusGroupTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "1xx").Add(float64(stats.Status1xx))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "200").Add(float64(stats.Status200))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "204").Add(float64(stats.Status204))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "206").Add(float64(stats.Status206))
+				m.StatusGroupTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "2xx").Add(float64(stats.Status2xx))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "301").Add(float64(stats.Status301))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "302").Add(float64(stats.Status302))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "304").Add(float64(stats.Status304))
+				m.StatusGroupTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "3xx").Add(float64(stats.Status3xx))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "400").Add(float64(stats.Status400))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "401").Add(float64(stats.Status401))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "403").Add(float64(stats.Status403))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "404").Add(float64(stats.Status404))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "416").Add(float64(stats.Status416))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "429").Add(float64(stats.Status429))
+				m.StatusGroupTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "4xx").Add(float64(stats.Status4xx))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "500").Add(float64(stats.Status500))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "501").Add(float64(stats.Status501))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "502").Add(float64(stats.Status502))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "503").Add(float64(stats.Status503))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "504").Add(float64(stats.Status504))
+				m.StatusCodeTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "505").Add(float64(stats.Status505))
+				m.StatusGroupTotal.WithLabelValues(serviceID, serviceName, datacenter, domain, "5xx").Add(float64(stats.Status5xx))
```

Comment:
`OriginOffload` and `EdgeHitRatio` are missing from this list.

Reply:
I left those off intentionally since they are calculated in the real-time API and could be calculated from the counters in promql. I can add them as gauges if you think they are useful.

Reply:
added.